google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -480,6 +501,15 @@ def deploy_to_local_endpoint(
         envs = {env.name: env.value for env in self.serving_container_spec.env}
         ports = [port.container_port for port in self.serving_container_spec.ports]
 
+        if gpu_count and gpu_device_ids:
```

Comment:
This logic is already contained in LocalEndpoint. Do we need to duplicate it here?

Reply:
Yes.